kolibri/core/__init__.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+"""TODO: Write something about this module (everything in the docstring
+enters the docs)
+
+.. moduleauthor:: Learning Equality <info@learningequality.org>
```

Comment:
Might be redundant to include this directive everywhere, plus in the case where we collaborate on modules it might not be clear who the single author is.